server/github.go

```diff
@@ -168,3 +171,54 @@ func (s *Server) checkUserPermission(user, repoOwner string) bool {
 
 	return true
 }
+
+func (s *Server) createRefWithPrefixFromPr(pr *model.PullRequest, prefix string) {
+	client := NewGithubClient(s.Config.GithubAccessToken)
+	_, _, err := client.Git.CreateRef(
+		context.Background(),
+		pr.RepoOwner,
+		pr.RepoName,
+		&github.Reference{
+			Ref: github.String("refs/heads/" + prefix + strconv.Itoa(pr.Number)),
```

Comment:
`/ref/heads/PRNUMBER`is the correct reference? i thought is like `refs/pull/PRNUMBER/head`

Reply:
According to the api docs (https://developer.github.com/v3/git/refs/#create-a-reference), this is the destination reference, which will be created. It fetches the PR commit via the given pr.Sha